src/sentry/static/sentry/app/views/performance/transaction_summary/table.tsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import {Location} from 'history';
+import styled from '@emotion/styled';
+
+import {Organization} from 'app/types';
+import space from 'app/styles/space';
+import {assert} from 'app/types/utils';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import {Panel} from 'app/components/panels';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import Link from 'app/components/links/link';
+import {TableData, TableDataRow, TableColumn} from 'app/views/eventsV2/table/types';
+import HeaderCell from 'app/views/eventsV2/table/headerCell';
+import EventView from 'app/views/eventsV2/eventView';
+import SortLink from 'app/views/eventsV2/sortLink';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import {getFieldRenderer, MetaType, getAggregateAlias} from 'app/views/eventsV2/utils';
+import {
+  generateEventSlug,
+  eventDetailsRouteWithEventView,
+} from 'app/views/eventsV2/eventDetails/utils';
+
+import {
+  TableGrid,
+  GridHead,
+  GridRow,
+  GridBody,
+  GridHeadCell,
+  GridBodyCell,
+  GridBodyCellNumber,
+} from '../styles';
+
+type Props = {
+  eventView: EventView;
+  location: Location;
+  organization: Organization;
+
+  isLoading: boolean;
+  tableData: TableData | null | undefined;
+};
+
+class SummaryContentTable extends React.Component<Props> {
+  renderHeader = () => {
+    const {eventView, tableData} = this.props;
+
+    const tableDataMeta = tableData && tableData.meta ? tableData.meta : undefined;
+
+    const columnOrder = eventView.getColumns();
+
+    const generateSortLink = () => undefined;
+
+    return columnOrder.map((column, index) => (
+      <HeaderCell column={column} tableData={tableData} key={index}>
```

Comment:
I would be nice to revisit this and figure out how to consolidate this table code with discover results. I feel like we can get to an API where the table wrapper has options for resizing/sorting/editing that enable a dynamic or static table to be created.